terra-base-jupyter/custom/test/test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+jupyter==1.0.0
+nose==1.3.7
+requests-mock==1.6.0
```

Comment:
... how does this relate to what is in `pyproject.toml`?

Reply:
It does not, for now I am still debating wether or not we need this (this is the leanest possible amount of legacy code that I am moving over from our current terra-docker process just in case).
If we end up using it, then I will streamline the testing and add this to the dev section of our `pyproject.toml`

Reply:
Actually, I already moved it 😅 so might as well delete this file altogether, thanks!